web/geo/types.ts

```diff
@@ -21,13 +21,23 @@ export interface ZoneConfig {
     productionPerModeForecast?: string;
     productionPerUnit?: string;
   };
-  [key: string]: any;
+  [key: string]:
+    | undefined
+    | string[]
+    | number[][]
+    | string
+    | boolean
+    | { [key: string]: string };
```

Comment:
Haha, at this point I'm not sure it's better than any 🤣

Reply:
Agreed but at least ESLint don't complain 😅

Reply:
We could add this somewhere 😆 
```
export interface Any {
  // Primitive types
  string: string;
  number: number;
  boolean: boolean;
  null: null;
  undefined: undefined;
  symbol: symbol;
  bigint: bigint;

  // Object types
  object: object;
  array: any[];
  tuple: [any, any];
  function: (...args: any[]) => any;

  // Union type
  union: string | number | boolean;

  // Intersection type
  intersection: { a: string } & { b: number };

  // Literal types
  stringLiteral: "literal";
  numberLiteral: 42;
  booleanLiteral: true;

  // Enum
  enum: "A" | "B" | "C";

  // Generic types
  generic: Array<any>;
  promise: Promise<any>;
  map: Map<any, any>;
  set: Set<any>;

  // Class
  class: { new (...args: any[]): any };

  // Recursive type
  recursive: Any;

  // Optional property
  optional?: any;

  // Index signature for additional properties
  [key: string]: unknown;
}
```